### ISSUE TITLE
Update benchmark submodule to build on ppc64

### ISSUE THREAD

**Issue by mehrdada** [google/protobuf#3717]
The `google/benchmark` submodule is pinned to an older version which has some build issues with PowerPC. Apparently this commit fixes the issue https://github.com/google/benchmark/commit/fef203bd021bad019421abf5785a4ae89bd6ea15 and the v1.2 of `google/benchmark` is already released with the fix out.

A gRPC user brought this to our attention and I am going to try updating our submodule dependency to `v1.2`. It might be good for `protobuf` to do the same to avoid a similar issue: https://github.com/grpc/grpc/issues/12694

**Comment by mehrdada** [comment]
(update: I am not sure, but perhaps we need to wait for https://github.com/google/benchmark/issues/449 to get fixed first)

**Comment by mandlebug** [comment]
@pherl 

I am building Protobuf on a ppc64 machine and I still have the build break from the `google/benchmark` submodule.  The commit that was merged in #3822 is in the log but it looks like its empty ( `git show` shows the commit info + msg but no files were changed). 

The gitlink is pointing to a commit from Wed May 11 2016: 
```
git ls-tree master third_party/benchmark/
160000 commit 360e66c1c4777c99402cf8cd535aa510fee16573  third_party/benchmark
```
which is way before the commit that needed to be picked up to fix the Power build.

Would you like me to open a new issue for this, or can this one be reopened to address this?
